testapp-cordova/package.json

```diff
@@ -27,12 +27,12 @@
         "chalk": "^4.1.2",
         "cordova-android": "^13.0.0",
         "cordova-ios": "^7.1.1",
-        "cordova-powerauth-mobile-sdk": "file:../build/cdv",
+        "cordova-powerauth-mobile-sdk": "file:../build/cdv/cordova-powerauth-mobile-sdk-3.0.0.tgz",
         "dotenv": "^16.4.5",
         "esbuild": "^0.23.1",
         "gulp": "^5.0.0",
         "gulp-replace": "^1.1.4",
-        "powerauth-js-test-client": "^0.2.12",
+        "powerauth-js-test-client": "file:../../powerauth-js-test-client/powerauth-js-test-client-0.2.13.tgz",
```

Comment:
Where 0.2.13?

Reply:
Right, wairing for approval - https://github.com/wultra/powerauth-js-test-client/pull/10